Spec/Tests/RealtimeClientConnectionTests.swift

```diff
@@ -2772,7 +2772,7 @@ class RealtimeClientConnectionTests: XCTestCase {
         }
     }
 
-    // FIXME: Fix flaky presence tests and re-enable. See https://ably-real-time.slack.com/archives/C030C5YLY/p1.2.182436085700
+    // FIXME: Fix flaky presence tests and re-enable. See https://ably-real-time.slack.com/archives/C030C5YLY/p1623172436085700
```

Comment:
I think this is an accident

Reply:
Did you see this commit @maratal? https://github.com/ably/ably-cocoa/pull/1558/commits/d3945fc37b22aaafa29fac66ba94a9df6e44851e